src/scripts.js

```diff
@@ -64,15 +102,15 @@ let sleepInfoQualityAverageAlltime = document.querySelector('#sleep-info-quality
 let sleepInfoQualityToday = document.querySelector('#sleep-info-quality-today');
 let sleepMainCard = document.querySelector('#sleep-main-card');
 let sleepUserHoursToday = document.querySelector('#sleep-user-hours-today');
-let sortedHydrationDataByDate = user.ouncesRecord.sort((a, b) => {
-  if (Object.keys(a)[0] > Object.keys(b)[0]) {
-    return -1;
-  }
-  if (Object.keys(a)[0] < Object.keys(b)[0]) {
-    return 1;
-  }
-  return 0;
-});
+// let sortedHydrationDataByDate = user.ouncesRecord.sort((a, b) => {
```

Comment:
I'm currently working on replacing all the data that was hardcoded in, some of the things are bit tangled. Goodluck! I'll push mine up soon so you can see a little bit of what I've done.